scripts/build_dockerhub_image.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# This script is used to create the docker image we push to dockerhub.
+# This requires that you have access to the dsluiuc dockerhub page. 
+# Currently, only Drake and Samarth have access to this.
+# Usage: sh scripts/build_dockerhub_image.sh
+
+docker login
+docker build . --target test-image --tag dsluiuc/honeybadgermpc-test-image
+docker push dsluiuc/honeybadgermpc-test-image:latest
```

Comment:
I don't think this will work...

What happens when two people update the code and want to upload different images to the same endpoint? One of them might pick up the other version..